doc/user-guide/03-configuration.adoc

```diff
@@ -93,7 +93,6 @@ The default boot option of the created ISO is boothd / "boot from first harddisk
 e.g. because you integrate REAR into some automation process, you can change the default using
 _ISO_DEFAULT={manual,automatic,boothd}_
 
-
```

Comment:
That removal of the empty line in doc/user-guide/03-configuration.adoc
is not needed here and may be removed from this pull request changes
so that there is not any change of doc/user-guide/03-configuration.adoc

Reply:
Sorry for that! Should be fine now.

Reply:
Thank you.
Now the changes look perfectly clean and straightforward.